File: CoffeeBack/src/main/java/br/ufsm/csi/coffee/model/Usuario.java
package br.ufsm.csi.coffee.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Usuario {
    private String nome;
    private String cpf;
    private String login;
    private String senha;
    private String token;
    private Funcao funcao;
}

File: CoffeeFront/src/app/login/Usuario.ts
import { Funcao } from "./funcao";

export class Usuario{
    nome?: string;
    login?: string;
    cpf?: string;
    senha?: string;
    token?: string;
    funcao?: Funcao;

    constructor(nome?: string, login?: string, cpf?: string, senha?:string, token?: string, funcao?: Funcao){
        this.cpf = cpf;
        this.nome = nome;
        this.login = login;
        this.senha = senha;
        this.token = token;
        this.funcao = funcao;
    }
}
